sdk/build.gradle

```diff
@@ -102,14 +102,14 @@ android {
             buildConfigField "String", "BASE_URL", "\"https://testaka4.sogei.it/v1/dgc/\""
             buildConfigField "String", "SERVER_HOST", "\"testaka4.sogei.it\""
             buildConfigField "String", "CERTIFICATE_SHA", "\"sha256/R0d+cI8vTcJ3sCbpfQCH0OmdBbulPH3deYhngzOqJVA=\""
-            buildConfigField "String", "SDK_VERSION", "\"1.1.6\""
+            buildConfigField "String", "SDK_VERSION", "\"1.1.7\""
```

Comment:
Ciao @Light2288 

Considerata la pubblicazione della storelease 1.3.0 con annesso bump di versione DGC-SDK a 1.1.8 (_ved. BuildConfig.java della storerelease su Google PlayStore / Huawei AppGallery_) + l'allineamento del codice DGC-SDK in storerelease con attuale branch develop, non dovrebbe esserci semmai il bump a 1.1.8 per buildType debug e release ?

`            buildConfigField "String", "SDK_VERSION", "\"1.1.8\""`

Reply:
ciao @rawmain, grazie per la segnalazione, corretto, chiudo questa pr e apro per il passaggio a 1.1.8